fix - test issues [ci skip]

ttodua opened the pull request:
(part 1)
these exchanges were failing in tests

ttodua commented on test-commonjs.cjs:
```diff
@@ -18,13 +18,13 @@ const symbol = 'BTC/USDT:USDT';
     try {
         // test cjs version
-        const exchange = new ccxt.bybit({});
+        const exchange = new ccxt.gate({});
```
bybit failed too frequently: https://app.travis-ci.com/github/ccxt/ccxt/builds/262664322#L2650

carlosmiei replied:
@ttodua I think this will fix it https://github.com/ccxt/ccxt/pull/17701

ttodua replied:
ah, havent seen it. ok reverted

ttodua replied:
btw, it is worth to have bybit there (which needs to have a proxy)? maybe we could replace it with non proxy needed one. like gate or other.

carlosmiei replied:
i think it is ok, and at least if the proxy stops working we can easily spot it

ttodua commented:
@carlosmiei passes build.

ttodua commented on ts/src/lbank2.ts:
```diff
@@ -309,6 +309,8 @@ export default class lbank2 extends Exchange {
                 '5s': true,
             };
+            const amountPrecision = this.parseNumber (this.parsePrecision (this.safeString (market, 'quantityAccuracy')));
+            const contractSize = amountPrecision;
```
lbank had same size as precision

carlosmiei commented:
@ttodua question, if these exchanges were failing, shouldn't we remove them from `skip-tests` now?

ttodua commented:
> @ttodua question, if these exchanges were failing, shouldn't we remove them from `skip-tests` now?

no, these things I've addressed were not failing in current master, but they were failing in my another upcoming stricter-tests PR. so, in-advance I've fixed these cases.

ttodua commented:
but don't worry, soon after we complete the tests for rest (some few days ahead still) i'll be revising the 'skips' dictionary from again.

ttodua commented:
let me know if I should change smth here.

carlosmiei commented:
LGTM ✅
